packages/kuma-gui/src/app/services/routes.ts

```diff
@@ -74,13 +63,7 @@ export const routes = (can: Can) => {
                     path: 'overview',
                     name: 'mesh-multi-zone-service-detail-view',
                     component: () => import('@/app/services/views/MeshMultiZoneServiceDetailView.vue'),
-                    children: [
-                      {
-                        path: ':dataPlane',
-                        name: 'mesh-multi-zone-service-data-plane-summary-view',
-                        component: () => import('@/app/data-planes/views/DataPlaneSummaryView.vue'),
-                      },
-                    ],
+                    children: dataPlanes().summary('mesh-multi-zone-service'),
```

Comment:
Just wanted to make a note here to share thinking/ideas. Just a note for the future nothing to be done now.

---

Eventually I would like this sort of stuff to be done from The Outside i.e. something like:

```js
// probably from within main.ts, or possibly `module-name/index.ts`
mount(summaryRoutes, 'mesh.dataplanes.summary')
mount(summaryRoutes, 'mesh.service.dataplanes.summary')
```

i.e. "mount these routes at this point in the routing tree please"

Reply:
Definitely that would be a great to have 👍 🙂